docs/policy/balloons.md

```diff
@@ -76,6 +76,13 @@ Balloons policy parameters:
 - `ReservedPoolNamespaces` is a list of namespaces (wildcards allowed)
   that are assigned to the special reserved balloon, that is, will run
   on reserved CPUs. This always includes the `kube-system` namespace.
+- `AllocatorTopologyBalancing` affects selecting CPUs for new
+  balloons. If `true`, new balloons are created using CPUs on
+  package/die/NUMA with most free CPUs. This optimizes for performance
+  and inflating balloons within the same package/die/NUMA. The default
```

Comment:
```suggestion
  package/die/NUMA with most free CPUs. This optimizes the performance
  and inflating balloons within the same package/die/NUMA. The default
```

Reply:
I ended up removing optimizing for power or performance completely, and explained in slightly lower level what this means in terms of balloon placement.